src/java.desktop/share/classes/javax/swing/border/LineBorder.java

```diff
@@ -24,6 +24,8 @@
  */
 package javax.swing.border;
 
+import com.sun.java.swing.SwingUtilities3;
```

Comment:
Let's be consistent: it should go after `java.*` and `javax.*` packages.

Either there should be a blank line before `com.*` or `sun.*` imports or there shouldn't be.